### PR TITLE
Updating english docs to default to American spellings

### PR DESCRIPTION
**Pull request by puneet-ekline**
### Description

Updating the English docs to use the US-based spellings. 
Reference: https://kubernetes.io/docs/contribute/style/style-guide/#language

More details about EkLine.io [here](https://ekline.atlassian.net/wiki/external/MDJmNWI3MjBkMzE4NGQyMzhlYjkxNjUwNDVhYjBlOGQ)

### CODE REVIEW

File: content/en/docs/reference/instrumentation/metrics.md
Comment (dipesh-rawat) [inline content/en/docs/reference/instrumentation/metrics.md:1]: This [Kubernetes Metrics Reference](https://kubernetes.io/docs/reference/instrumentation/metrics/) page is [auto-generated](https://kubernetes.io/docs/reference/instrumentation/metrics/#auto-generated-edit-notice).

_For more information on generating Metrics page, please refer to the documentation here: https://kubernetes.io/docs/contribute/generate-ref-docs/metrics-reference/._

File: content/en/docs/reference/config-api/kubelet-config.v1beta1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/config-api/kubelet-config.v1beta1.md:1]: This reference page for  "_Kubelet Configuration (v1beta1)_" is [automatically generated](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#auto-generated-edit-notice).

File: content/en/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md:1]: This reference page for  "_PersistentVolume_" is [automatically generated](https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1/#auto-generated-edit-notice).

File: content/en/docs/reference/kubernetes-api/config-and-storage-resources/volume.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/config-and-storage-resources/volume.md:1]: This reference page for  "Volume" is [automatically generated](https://kubernetes.io/docs/reference/kubernetes-api/config-and-storage-resources/volume/#auto-generated-edit-notice).

File: content/en/docs/reference/kubernetes-api/extend-resources/custom-resource-definition-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/extend-resources/custom-resource-definition-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/policy-resources/network-policy-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/policy-resources/pod-disruption-budget-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/policy-resources/pod-disruption-budget-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/service-resources/service-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/service-resources/service-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/daemon-set-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/deployment-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/deployment-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/horizontal-pod-autoscaler-v2.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/job-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/job-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/pod-v1.md:1]: This reference page is **_automatically generated_**.

File: content/en/docs/reference/kubernetes-api/workload-resources/stateful-set-v1.md
Comment (dipesh-rawat) [inline content/en/docs/reference/kubernetes-api/workload-resources/stateful-set-v1.md:1]: This reference page is **_automatically generated_**.

**Review by dipesh-rawat — COMMENTED** [review]
Thanks for your contribution! It seems that this PR is trying to update content on some pages that are **_auto-generated._** Instead, we should address the underlying issue by fixing the generator code or the API definition that the generator relies on. If any changes are needed for these pages.

_For more information on generating reference pages, please refer to the documentation here: https://kubernetes.io/docs/contribute/generate-ref-docs/kubernetes-components/._

**Comment by dipesh-rawat** [comment]
/hold

Refer https://github.com/kubernetes/website/pull/47774#pullrequestreview-2278698049
Instead of modifying some of the generated pages directly, it would be more effective to either regenerate all the reference documentation incorporating upstream fixes or consider excluding the auto-generated pages from this pull request altogether.

_Feel free to unhold this once either of the above actions has been taken._

**Comment by puneet-ekline** [comment]
Hi @dipesh-rawat  thanks for the feedback. I'll take a look at these autogenerated pages. Will address them tomorrow.

**Comment by natalisucks** [comment]
Hi @puneet-ekline – thank you for your interest in contributing to Kubernetes.

My name is Natali Vlatko, and I'm one of the co-chairs of SIG Docs, the technical maintainer group that owns the Kubernetes documentation and website. I wanted to let you know that the tech leads and chairs of our SIG have decided to close this PR.

While some AI tooling is used by contributors to assist with their upstream work, your association with this paid product and your attempt to use Kubernetes as advertising for its capabilities isn't in the best interest of the project. While we appreciate the transparency you've used in your PR description about the tool, we would have expected a conversation with project leads first before your attempt to spruik your product by why of this contribution. Also, as can be seen from Dipesh's review of your work, the tooling you've used hasn't been able to distinguish between our auto-generated pages, leaving too much room for error when using this product.

In light of this (and other work by you with this tool that we'll also close), SIG Docs will be working on more generalised AI tooling policy moving forward, and we invite you to keep up to date with this by joining the Kubernetes Slack or attending our bi-weekly SIG Docs community meetings. In the meantime, please visit https://kubernetes.io/docs/contribute/docs/ to understand how to best contribute to Kubernetes for future work.

/close

**Comment by puneet-ekline** [comment]
Thanks, @natalisucks, for the update. I would love to be part of the SIG Docs meetings and help craft a solution that would work for Kubernetes.

**Pull request closed without merging**